gatsby-config.js

```diff
@@ -8,10 +8,10 @@
 
 module.exports = {
   siteMetadata: {
-    title: 'React: A JavaScript library for building user interfaces',
+    title: 'React: JavaScript-бібліотека для створення користувацьких інтерфейсів',
```

Comment:
```suggestion
    title: 'React – JavaScript-бібліотека для створення користувацьких інтерфейсів',
```

Напевно, повинно бути тире замість двокрапки?